precompiles/wasmd/wasmd.go

```diff
@@ -110,8 +110,8 @@ func (p Precompile) RunAndCalculateGas(evm *vm.EVM, caller common.Address, input
 	if err != nil {
 		return nil, 0, err
 	}
-	gasMultipler := p.evmKeeper.GetPriorityNormalizer(ctx)
-	gasLimitBigInt := new(big.Int).Mul(new(big.Int).SetUint64(suppliedGas), gasMultipler.RoundInt().BigInt())
+	gasMultiplier := p.evmKeeper.GetPriorityNormalizer(ctx)
+	gasLimitBigInt := sdk.NewDecFromInt(sdk.NewIntFromUint64(suppliedGas)).Mul(gasMultiplier).RoundInt().BigInt()
```

Comment:
it was not changed in aspect of logic. I just thought performing `RoundInt()` later may be better to reduce rounding error.